client/state/checklist/reducer.js

```diff
@@ -26,7 +26,7 @@ const moduleTaskMap = {
 	videopress: CHECKLIST_KNOWN_TASKS.JETPACK_VIDEO_HOSTING,
 };
 
-const items = withSchemaValidation( itemSchemas, ( state = {}, action ) => {
+const items = withSchemaValidation( itemSchemas, ( state = null, action ) => {
```

Comment:
I'd change `getSiteTaskList()` to make it more obvious that `getSiteChecklist()` can be `null`, because while Lodash's `get()` will handle the `null` case, it's not immediately clear and it could break in an eventual migration away from Lodash.

Reply:
I fixed that by migrating the one relevant usage of `_.get`, in `getSiteTaskList`. Does that solve the problem?

Reply:
It does. Optional chaining already hints that the value may be nullish, while `_.get()` doesn't. Thanks!